src/PresentationalComponents/ZeroState/AppZeroState.js

```diff
@@ -122,7 +124,8 @@ NewAppZeroState.propTypes = {
     customTitle: propTypes.string,
     appId: propTypes.string,
     children: propTypes.any,
-    customFetchResults: propTypes.bool
+    customFetchResults: propTypes.bool,
+    customSection: propTypes.any
```

Comment:
We should set the proptype here to be "node" in order to signal that a React component is expected.